### DISCUSSION TITLE
is it technically possible connect two flows together?

### DISCUSSION THREAD

**Discussion by dndudgk1519** [postmanlabs/postman-flows#391] [Q&A]
hi i'm newbie about postman. and i'm just studying it.
i think it's super fool question. but is it possible connect each flows together?
like 
if i have to test AB flows
A_flows : A part of AB
    and
B_flows : B part of  AB
i want to test each flows individually and merge it last
thanks

**Comment by samay-v** [comment]
I think it's a great question @dndudgk1519! Today you can't connect flows inside of a flow. But this is an idea which the team and I have been tinkering for a while. So possibly in the future you'll be able to do this as well

**Reply by dndudgk1519** [reply]
thank you sososososo much :)
